docs/translation/index.md

```diff
@@ -205,6 +205,16 @@ Typography is different in different languages and thus must be translated along
     {ts}Event Total:{/ts}
     ```
 
+### Consider leaving existing bad uses of ts unchanged when the intended functionality is working
+
+Because of demands on translators' time and the number of steps in the end-to-end translation process itself, it is often better to leave existing bad instances of `ts` unchanged unless they are preventing users from completing tasks. For example a link that is malformed html and so doesn't go anywhere when clicked should be changed. An example that you should consider leaving as is would be:
+
+!!! failure "Consider leaving this as-is"
+
+    ```smarty
+    {ts 1="https://www.example.org"}This is a block of text where the wording is fine and the link works but the <a href="%1" target="_blank">link attributes</a> are wrong. But it's not preventing anyone completing their work tasks and isn't confusing to users.{/ts}
+    ```
```

Comment:
It took me a while to understand this example. I think by "link attributes are wrong" you mean like `target="_blank"` needs to be changed.

If this were a real-world example, as a developer this one would be a tough call for me. Leaving it alone is an option, but if it needed to be changed I'd try to make a swift change that avoids future pain, so I'd probably change the code to:

    ```smarty
    {ts 1='href="https://www.example.org" target="_something_else"'}This is a block of text where the wording is fine and the link works but the <a %1>link attributes</a> are wrong. But it's not preventing anyone completing their work tasks and isn't confusing to users.{/ts}
    ```

That way attributes can be updated in the future without changing the gettext string. While I was at it I'd carefully review the wording, since the string is impacted by this change anyway, and make any nitpicky tweaks that we normally wouldn't bother translators with.

Reply:
I mean everything is working from the user point of view and the wording is fine, but the target should be in the ts parameter and not have one attribute in the ts and one in the `a`. But when I did that kind of cleanup in real life see paragraph 3 at https://github.com/civicrm/civicrm-core/pull/16381#issuecomment-578422899. So let's make a policy either don't clean these up, or take the hit and do clean it up.

Reply:
I see what you mean by "wrong" not being clear. I've made an update to try to clarify the clarification.